modules/sdk/project-templates-freemarker-portlet/src/main/resources/archetype-resources/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -0,0 +1,2 @@
+.portlet-freemarker {
```

Comment:
We don't have this same thing in the other portlets.  I can see its a nice way for developers to add this wrapper to their portlets.  We should probably add it for other portlets `mvc-portlet`, `spring-mvc-portlet`, maybe even with `portlet`?